rfcs/text/0000-rfc-process-updates.md

```diff
@@ -0,0 +1,130 @@
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+Over the last year since the original RFC process was accepted, the DCP project has grown both in size and complexity.
+As of this writing, 11 RFCs have been accepted in addition to many charters. Not so unexpectedly, there have been pain
+points along the way of which we try to address a few via edits to the RFC process.
+
+1 . Limited states of an RFC:
+
+Currently the states of an RFC is one of "rfc-community-review", "rfc-oversight-review", 
+"rfc-approved", or "rfc-declined." In reality, there have been cases where an RFC is no longer implemented (i.e. 
+"deprecated") or an RFC has been withdrawn due to either incompleteness or having been published too early. 
+Additionally, with the encouraged behavior of submitting draft pull requests prior to RFC readiness for review, there
+is no way to clearly designate the fact that the RFC should not be reviewed at all until the PR is no longer a draft.
+
+2 . Minimum review period is too long:
+
+The process today requires a minimum of 2 weeks in DCP Community Review followed by 1 week of DCP Oversight Review.
+While the intention of the "Community" review was to receive feedback not from other DCP team members, but rather the 
+scientific community, in practice, most RFCs are reviewed as a free-for-all during the community review period followed
+by one week of radio silence since the conflicts have been resolved. 
+
+3 . Uncertainty around "trivial" edits to published RFCs:
+
+The original RFC process does not explicitly mention that minor edits (i.e. typos, re-wordings, etc.) are free to be
+pushed whenever needed. 
+
+4 . Lack of followup post RFC publication:
+
+Today, there is no requirement that RFCs must be related to any existing ticket in Zenhub which is usually a proxy for
+determining critical work to be done in a quarter. This means that RFCs are published with no clear roadmap to
+completion which also weakens the implication of the RFC. 
+
+### User Stories
+
+*Share the [User Stories](https://www.mountaingoatsoftware.com/agile/user-stories) motivating this RFC.*
+
+## Detailed Design
+
+We propse the following edits to the RFC process to address 1-4 above.
+
+1 . Limited states of an RFC:
+
+We proposed adding the tags "pre-rfc-do-not-comment", "rfc-deprecated", "rfc-paused", and "rfc-withdrawn" as new tags with the following 
+semantics:
+
+"pre-rfc-do-not-comment" : This indicates that the authors are still actively working on the RFC, that the RFC is in a
+draft state, and readers should **not** comment on the RFC yet. 
+
+"rfc-deprecated" : This indicates that the design is no longer implemented or executed in the DCP and has been usurped
+by a newer design or process.
+
+"rfc-paused" : This indicates that an RFC has been paused for further refinement. This roughly implys that an RFC has
```

Comment:
Is the rationale for pausing instead of closing that comments would be maintained? Is there guidance for withdrawing paused RFCs to avoid a glut of open PRs that aren't actionable?

Reply:
So pausing and closing a PR are mutually exclusive and I haven't detailed any guidelines yet on closing since Dave and I still need to flesh this out. Tags can be applied to any PR whether open or closed and allows for ease of searching for a state of a PR.

Sorry, there are lots of things here that still need fleshing out :D Still under active development, but thank you for the early feedback!

Reply:
As an example, I _paused_ the original Planning RFC after the Oversight Committee requested a roadmap from the Project Leads to understand its implications on planning and execution. Then I _withdrew_ the original RFC before Oversight Review because it was clear that the planning model would need to be redesigned and adapted to the availability of a long-term roadmap.

Reply:
Closing this thread as I believe it has been addressed, but please feel free to re-open if not!